linera-views/src/dynamo_db.rs

```diff
@@ -73,9 +73,12 @@ const KEY_VALUE_ATTRIBUTE: &str = "item_key, item_value";
 /// See https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/ServiceQuotas.html
 const MAX_VALUE_BYTES: usize = 409600;
 
+/// This limit guarantees that the values seen by DynamoDb are under 400 kB = 409600 bytes.
+const MAX_VALUE_SIZE: usize = 409000;
```

Comment:
Are there deliberately 600 bytes missing? What are they reserved for?

Reply:
Maybe we should find out how it works exactly at some point (otherwise we never know if even 600 bytes are enough leeway). Could be a separate issue, of course.

Reply:
Ok, I unified this, now remove those 600.
I will do some offline checks about that issue and see if there is anything wrong.

Reply:
Well of course, changing created the problem, exactly as I was afraid of:
        error: "Storage operation error in DynamoDB: Error { code: \"ValidationException\", message: \"Item size has exceeded the maximum allowed size\", request_id: \"01545aa4-58d0-4b6b-9429-be427b6dda98\" }",

Let me remove again the 600, and create an issue.